static/src/javascripts-legacy/projects/common/modules/experiments/tests/generic-email-variants.js

```diff
@@ -0,0 +1,68 @@
+define([
+    'bean',
+    'fastdom',
+    'qwery',
+    'lib/$',
+    'lib/config',
+    'lib/fastdom-promise'
+], function (
+    bean,
+    fastdom,
+    qwery,
+    $,
+    config,
+    fastdomPromise
+) {
+    return function (props) {
+        this.id = props['id'];
```

Comment:
Using square bracket notation with string literals for accessing object properties is stylistically a bit weird (https://github.com/airbnb/javascript#properties) - best to stick with dot notation unless you're using a variable as the key